app.py

```diff
@@ -348,5 +348,7 @@ def build_content(langcode):
 @app.route("/", methods=["GET", "POST"], defaults={"path": ""})
 @app.route("/<path>", methods=["GET", "POST"])
 def index(path):
+    if path == '' and 'langcode' in request.args:
+        return redirect(url_for('index', path=request.args['langcode']))
```

Comment:
Or, if you want to use Python 3.8’s “walrus” operator:
```suggestion
    if path == '' and langcode := request.args.get('langcode'):
        return redirect(url_for('index', path=langcode))
```